projects/design-angular-kit/src/lib/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  Component,
+  ChangeDetectionStrategy,
+  ElementRef,
+  Inject,
+  Input,
+  Optional,
+  ViewEncapsulation
+} from '@angular/core';
+
+const BG_DEFAULT = undefined;
+const BG_DANGER = 'bg-danger';
+const BG_INFO = 'bg-info';
+const BG_WARNING = 'bg-warning';
+const BG_SUCCESS = 'bg-success';
```

Comment:
Inserire i colori in un file generico così da poterli riutilizzare in altre componenti.